config/base.yml

```diff
@@ -120,8 +120,9 @@ script:
   # BEHAT
   - if [[ $BEHAT_TEST ]]; then vendor/bin/behat $BEHAT_SUITE; fi
 
-  # PHPCS
+  # PHPCS and PHPSTAN
   - if [[ $PHPCS_TEST && -f phpcs.xml.dist ]]; then vendor/bin/phpcs; fi
+  - if [[ $PHPCS_TEST && -f phpstan.neon.dist ]]; then vendor/bin/phpstan analyse; fi
```

Comment:
What's the benefit of coupling it with phpcs? Shouldn't it be completely separate?